### ISSUE TITLE
Ubuntu 22.04 has no libglew2.1

### ISSUE THREAD

**Issue by XilinJia** [emeiri/ogldev#18]
It has libglew2.2 instead

**Comment by emeiri** [comment]
Yep, that script was written for the previous version of Ubuntu. I will need to make the script more flexible to support more versions. Thanks.

**Comment by emeiri** [comment]
Fixed. Thanks for the feedback!
